src/modules/customer/containers/HappyUserPage/components/WizardForm/utils/validate.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
this is not utils. Utils are service functions. Just move this file to the form level.